pages/user.js

```diff
@@ -17,12 +17,7 @@ import ClaimList from '../components/ClaimList';
 import MessageList from '../components/MessageList';
 
 import formDataUtils from '../utils/formDataUtils';
-import {
-  fetchUserInfo,
-  fetchClaims,
-  fetchMamMessages,
-  createClaim,
-} from '../utils/tangleidAPI';
+import tangleid from '../utils/tangleidSetup';
```

Comment:
`tangleidSetup` looks a bit tediously long. How about simple `setup`?

Reply:
The prefix `tangleid` is necessary that can help us distinguish which API is setup, since we already have `iotaSetup` that setup the IOTA API instance.